src/detection/DepthImagePolarHistDetector.cc

```diff
@@ -101,7 +101,17 @@ const std::vector<Obstacle> &DepthImagePolarHistDetector::detect(
         double phi = max_phi - (i * fixed_step) - (fixed_step / 2);
 
         Obstacle obs;
+
+        obs.bounding_box.tlc.x = histogram[i];
+        obs.bounding_box.tlc.y = (M_PI - depth_data->vfov) / 2 ;
```

Comment:
Indeed...
We really need proper names for those variables... we waste a lot of time because of silly confusions.